cmd/zpool/zpool_main.c

```diff
@@ -9252,6 +9252,15 @@ main(int argc, char **argv)
 	if ((strcmp(cmdname, "-?") == 0) || strcmp(cmdname, "--help") == 0)
 		usage(B_TRUE);
 
+	/*
+	 * Special case '-V|--version'
+	 */
+	if ((strcmp(cmdname, "-V") == 0) ||
+	    (strcmp(cmdname, "--version") == 0)) {
+		zfs_version_print();
+		return (0);
+	}
```

Comment:
Let's add the new option to the `zfs.8` and `zpool.8` man pages.  Something like this:

```diff
diff --git a/man/man8/zpool.8 b/man/man8/zpool.8
index 7ba8a5b..fcef3c2 100644
--- a/man/man8/zpool.8
+++ b/man/man8/zpool.8
@@ -35,7 +35,7 @@
 .Nd configure ZFS storage pools
 .Sh SYNOPSIS
 .Nm
-.Fl ?
+.Fl ?V
 .Nm
 .Cm add
 .Op Fl fgLnP
@@ -909,6 +909,11 @@ The following subcommands are supported:
 Displays a help message.
 .It Xo
 .Nm
+.Fl V -version
+.Xc
+Displays the ZFS version information.
+.It Xo
+.Nm
 .Cm add
 .Op Fl fgLnP
 .Oo Fl o Ar property Ns = Ns Ar value Oc
```

Reply:
Thank you for that, I've added to both zfs.8 and zpool.8